validator/testcases/regex.py

```diff
@@ -1516,6 +1516,36 @@ def tests(self):
             compat_type="error")
 
 
+@register_generator
+class Gecko35RegexTests(CompatRegexTestHelper):
+    """Regex tests for Gecko 35 updates."""
+
+    VERSION = FX35_DEFINITION
+
+    def tests(self):
+        box_object_subject = ("All interfaces derived from nsIBoxObject no "
+                              "longer exist.")
+        yield self.get_test(
+            r"\bnsI[\w\d]*BoxObject\b",
+            box_object_subject,
+            box_object_subject + " If you use them with "
+            "QueryInterface it's likely you just need to remove "
+            "the function call. See %s for more information."
+            % BUGZILLA_BUG % 979835,
+            log_function=self.err.warning,
+            compat_type="error")
+
+        iterator_link = (MDN_DOC % "Web/JavaScript/Reference/Global_Objects"
+                                   "/String/@@iterator")
+        yield self.get_test(
+            r"['\"]@@iterator",
```

Comment:
Ah, yeah I meant to match on both, must have forgotten. I thought about doing a `"""` then didn't because the one `\` was shorter. It's probably a good call though. `\b` doesn't seem to match quotes (or didn't make the test pass anyway).